appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java

```diff
@@ -48,6 +48,8 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.ExecutionException;
+import java.util.logging.Level;
```

Comment:
Ah yes, I branched off #257.   I've merged with the latest version of that, but can remake as a separate PR if you prefer.